lxd/network/driver_bridge.go

```diff
@@ -3047,7 +3047,7 @@ func (n *bridge) getExternalSubnetInUse() ([]externalSubnetUsage, error) {
 }
 
 // ForwardCreate creates a network forward.
-func (n *bridge) ForwardCreate(forward api.NetworkForwardsPost, clientType request.ClientType) error {
+func (n *bridge) ForwardCreate(forward *api.NetworkForwardsPost, clientType request.ClientType) error {
```

Comment:
Can we instead of modifying the input param change this function signature to return a net.IP of the requested listen address? 

In general having functions modify their input arguments, but only in certain scenarios, can lead to confusion - whereas always returning the listen IP separately would be more predictable IMHO. What do you think?

Reply:
I was aiming to change the network driver interface as little as possible but I agree. Will change.